cloud-common/src/main/resources/application-prod.yml

```diff
@@ -0,0 +1 @@
+cb.image.catalog.url: https://s3-eu-west-1.amazonaws.com/cloudbreak-info/v2-prod-cb-image-catalog.json
```

Comment:
Not public

Reply:
That will be available after the first release build.